src/common.rs

```diff
@@ -4,9 +4,10 @@ use std::path::PathBuf;
 
 #[derive(Clone, Debug, PartialEq)]
 pub struct Warning {
-    check_name: String,
+    pub check_name: String,
     line: LineEntry,
     message: String,
+    pub is_fixed: Option<bool>,
```

Comment:
```suggestion
    pub is_fixed: bool,
```
What do you think about using just `bool` type instead of `Option<bool>`?

Reply:
`None` is used when we are not fixing Warnings, in this case they are no `Fixed`, no `Unfixed`.

Reply:
> `None` is used when we are not fixing Warnings, in this case they are no `Fixed`, no `Unfixed`.

It seems to me, by default, an `Warning` cannot have such a state.
When created, it is not automatically fixed.
Correct me if it is not

Reply:
Yes, it is no automatically Unfixed, no Fixed. This is the point.
But I will study alternative approaches.

Reply:
There is no need in Option anymore.